src/herder/HerderSCPDriver.cpp

```diff
@@ -489,7 +489,45 @@ HerderSCPDriver::setupTimer(uint64_t slotIndex, int timerID,
     }
 }
 
-// core SCP
+// returns true if l < r
+// lh, rh are the hashes of l,h
+static bool
+compareTxSets(TxSetFramePtr l, TxSetFramePtr r, Hash const& lh, Hash const& rh,
+              LedgerHeader const& header, Hash const& s)
+{
+    if (l == nullptr)
+    {
+        return r != nullptr;
+    }
+    if (r == nullptr)
+    {
+        return false;
+    }
+    auto lSize = l->size(header);
+    auto rSize = r->size(header);
+    if (lSize < rSize)
+    {
+        return true;
+    }
+    else if (lSize > rSize)
+    {
+        return false;
+    }
+    if (header.ledgerVersion >= 11)
```

Comment:
For me it is a bit confusing that there are two condition on ledger version here, one hidden (inside l->size and r->size), and one visible here. I think it would be clearer here to explicitly use sizeTx or sizeOp depending on ledgerVersion

Reply:
At this layer what we compare are "transaction set sizes", ops or txs is an implementation detail
That said, I am renaming `size` for clarity